gradio/themes/utils.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+
+class Theme:
```

Comment:
I think `to_json` will make it much easier to share themes in the future. It's probably a good idea to have a way to serialize themes outside of pickle anyways.

Reply:
Let's figure out sharing themes in another PR?

Reply:
Outside of sharing broadly though, if I have a theme defined locally and I want to use it for several different demos, I’d be nice to have a way to serialize it to save it. We could use pickle but that doesn’t work as nice with git/version control.

Reply:
I’m good on doing it on this PR or a next one.